### PR TITLE
Component Updates - 29.07.2022

### PR DESCRIPTION
**Pull request by heoelri**
This PR contains some of the component-updates brought up by dependabot this morning. 

* some minor version updates for dotnet components
* bumps the build env for the alwayson ui container to `node:18.7.0`
* bumps grafana to `9.0.5` (yay!)
* bumps terraform from `3.15.1` to `3.16`

Successfully tested here:
* e2e: https://dev.azure.com/Mission-Critical/Online/_build/results?buildId=1800&view=results
* grafana: https://dev.azure.com/Mission-Critical/Online/_build/results?buildId=1801&view=results
* load gen: https://dev.azure.com/Mission-Critical/Online/_build/results?buildId=1802&view=results

### CODE REVIEW

**Comment by heoelri** [comment]
@msimecek as discussed.

**Comment by heoelri** [comment]
```console
/home/LogFiles/2022_07_29_pl0sdlwk000HWH_default_docker.log  (https://afintgrafana-eastu-app.scm.azurewebsites.net/api/vfs/LogFiles/2022_07_29_pl0sdlwk000HWH_default_docker.log)
2022-07-29T07:48:41.923470086Z logger=migrator t=2022-07-29T07:48:41.923214972Z level=error msg="alert migration failure: could not get migration log" error="failed to check table existence: pq: The public network access on this server is disabled. To connect to this server, use the Private Endpoint from inside your virtual network."

2022-07-29T07:49:04.821519251Z logger=migrator t=2022-07-29T07:49:04.821248136Z level=error msg="alert migration failure: could not get migration log" error="failed to check table existence: pq: The public network access on this server is disabled. To connect to this server, use the Private Endpoint from inside your virtual network."

2022-07-29T07:51:08.825533942Z logger=migrator t=2022-07-29T07:51:08.811055406Z level=error msg="alert migration failure: could not get migration log" error="failed to check table existence: pq: The public network access on this server is disabled. To connect to this server, use the Private Endpoint from inside your virtual network."

2022-07-29T07:51:23.263520299Z logger=migrator t=2022-07-29T07:51:23.263043472Z level=error msg="alert migration failure: could not get migration log" error="failed to check table existence: pq: The public network access on this server is disabled. To connect to this server, use the Private Endpoint from inside your virtual network."

2022-07-29T07:53:57.675635520Z logger=migrator t=2022-07-29T07:53:57.674579059Z level=error msg="alert migration failure: could not get migration log" error="failed to check table existence: pq: The public network access on this server is disabled. To connect to this server, use the Private Endpoint from inside your virtual network."
```

Grafana is not fully working yet. This needs to be fixed first.

File: src/infra/monitoring/grafana/terraform/stamps/app-service.tf [hunk 1]
```diff
@@ -27,7 +27,7 @@ resource "azurerm_linux_web_app" "appservice" {
   // these env variables are specific to postgres backend supported by grafana
   app_settings = {
     "GF_DATABASE_TYPE"     = "postgres"
-    "GF_DATABASE_HOST"     = "${each.key == "primary" ? azurerm_postgresql_server.pgprimary.name : azurerm_postgresql_server.pgreplica.name}.postgres.database.azure.com"
```
Comment (heoelri) [inline src/infra/monitoring/grafana/terraform/stamps/app-service.tf:30]: @nielsams had to switch to the privatelink dns name to make it work. Wonder why this was working before. Public access to the PGSQL is/was disabled all the time.

The secondary region can probably not reach the primary PGSQL (and vice versa).

**Comment by msimecek** [comment]
Is this all? I can see that `src/testing/userload-generator/AzureFunctions/RegionalLoadGenerator/package-lock.json` has changed, but there's no change in `src/testing/userload-generator/AzureFunctions/RegionalLoadGenerator/package.json`.

**Comment by heoelri** [comment]
> Is this all? I can see that `src/testing/userload-generator/AzureFunctions/RegionalLoadGenerator/package-lock.json` has changed, but there's no change in `src/testing/userload-generator/AzureFunctions/RegionalLoadGenerator/package.json`.

It wasn't me - it was dependabot.

**Comment by msimecek** [comment]
Ok, than it's perfect :D

**Review by msimecek — APPROVED** [review]

**Pull request merged**